### DISCUSSION TITLE
Accessing lookup from server-side

### DISCUSSION THREAD

**Discussion by dfaruque** [serenity-is/Serenity#6128] [Help] [answered]
In typescript, we have the following to get a lookup item by its id
`someRow.getLookup().itemById[key]`

Is there something similar for c#?

Actually, this is an old issue https://github.com/serenity-is/Serenity/issues/3656
But the solution mentioned there is a custom solution. Is there a built-in method of doing the same?

**Comment by VictorTomaili** [comment, marked as answer]
Nope, you can access the DB on the CSharp code. lookups are generated from the DB when requested from the javascript.
